### ISSUE TITLE
Duplicate "ghost" breakpoints 

### ISSUE THREAD

**Issue by sonictk** [x13pixels/remedybg-issues#151]
Hi:

I noticed an issue when re-attaching to processes with RemedyBG; it seems like there are instances where breakpoints that were previously saved do not show indicators in the source view when re-attaching to the same process, and you can end up adding duplicate breakpoints as well.

I've attached screenshots of what this looks like when it manifests:

![tmp0](https://user-images.githubusercontent.com/2504199/78644939-8ebff500-786b-11ea-895d-bd7206618a63.png)

If I try to add a breakpoint at the same line again, the breakpoint indicator then shows up in the source view, but it looks like a duplicate entry is created.
![tmp1](https://user-images.githubusercontent.com/2504199/78644977-a13a2e80-786b-11ea-9015-c9514da558e9.png)

**Comment by sonictk** [comment]
As an addendum, after I attach/detach from a process a couple of times (`Photoshop.exe`), I'm unable to add any more breakpoints in the source window. Whenever I try adding a new one, no breakpoint appears, and a new entry appears in the breakpoints tab that doesn't make any sense. I've attached a new screenshot that shows what I see. (A new entry is added each time I try to add a new breakpoint)

![image](https://user-images.githubusercontent.com/2504199/78756136-2aff0000-792f-11ea-826d-f17bfaed51d7.png)

I know that Photoshop basically loads/unloads the DLL each time I invoke my plugin to be run, I wonder if RemedyBG has some made some assumptions here that might be related to the issue?

**Comment by x13pixels** [comment]
That could definitely be the problem. Thanks for the additional information!

**Comment by x13pixels** [comment]
I believe I've fixed this in 0.3.0.3 but will leave it open until confirmed.

**Comment by sonictk** [comment]
It's somewhat fixed, but not entirely, Unfortunately I discovered a new issue (also present in 3.0.2, it looks like), which is that after the first time I run the code, I can no longer hit the breakpoint ever again on the second time I invoke it unless I detach from Photoshop, and then re-attach. Also, while at least now I'm able to add the breakpoint after detaching and re-attaching to Photoshop, duplicate entries still appear in the Breakpoints window. By checking/unchecking the "ghost" entries around repeatedly, I managed to crash remedybg once, but I'm unable to reproduce it reliably.

I recorded a new video to demonstrate what I'm seeing:

https://youtu.be/tgzY7EsqsXg

If you have a fairly recent copy of Photoshop (CC2017 and above) available, I'd be happy to provide source code/binaries to assist, if it would help you in debugging this issue.

(Bah, I just realized that the Windows Game bar capture only captured the RemedyBG window and not the desktop. Basically, when you see my cursor go off the screen, I'm interacting with Photoshop in order to invoke the code being run, or closing a GUI dialog that it pops up in order to allow execution to continue.)
